stages/test/test_erofs.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python3
+
+import os.path
+import subprocess
+from unittest import mock
+
+import pytest
+
+import osbuild.meta
+from osbuild.testutil.imports import import_module_from_path
+
+
+def make_fake_input_tree(tmpdir, fake_content: dict) -> str:
```

Comment:
I can totally imagine this being in `testutils` and being reused by more tests. Can be done in a follow-up/when needed.

Reply:
I like this idea! I will put it on my TODO :)